openstack/objectstorage/v1/containers/testing/requests_test.go

```diff
@@ -18,18 +19,17 @@ func TestContainerNames(t *testing.T) {
 	for _, tc := range [...]struct {
 		name          string
 		containerName string
+		expectedError error
 	}{
 		{
 			"rejects_a_slash",
 			"one/two",
+			v1.ErrInvalidContainerName{},
 		},
 		{
-			"rejects_an_escaped_slash",
-			"one%2Ftwo",
```

Comment:
The name validation is always run right before encoding in Gophercloud. Thus if the caller sent us an encoded slash, we can happily take it as it will be double-encoded and will not expand as a slash in Swift.